date/main.js

```diff
@@ -1,6 +1,10 @@
 // year: number
 function isLeapYear(year) {
     //TODO
+    label:
+    if (year % 400 == 0) return False, break label;
```

Comment:
why do you need a break after a return statement?

Reply:
resolved